### ISSUE TITLE
Uncaught TypeError: Cannot read property 'classList' of null

### ISSUE THREAD

**Issue by mikecluck** [Osmose/speedkills#1]
I get this error when viewing the Settings panel instead of an editor panel.

**Atom Version**: 0.206.0
**System**: Microsoft Windows 8.1
**Thrown From**: [speedkills](https://github.com/osmose/speedkills) package, v0.1.1
### Stack Trace

Uncaught TypeError: Cannot read property 'classList' of null

```
At C:\Users\Mike\.atom\packages\speedkills\lib\speedkills.coffee:87

TypeError: Cannot read property 'classList' of null
  at Object.module.exports.stop (C:\Users\Mike\.atom\packages\speedkills\lib\speedkills.coffee:87:16)
  at Object.module.exports.checkRate (C:\Users\Mike\.atom\packages\speedkills\lib\speedkills.coffee:67:8)
  at C:\Users\Mike\.atom\packages\speedkills\lib\speedkills.coffee:28:34

```
### Commands

```
     -1:01.7.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.theme-steam-pirate-syntax.theme-steam-pirate-ui)
  2x -0:18.9.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:17.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "steam-pirate-ui",
      "steam-pirate-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
api-docs, v0.0.1
file-icons, v1.5.7
javascript-snippets, v1.0.0
language-fxi-2015, v1.0.0
language-kxi-2015, v1.0.2
language-uvu-asm, v0.4.0
speedkills, v0.1.1
steam-pirate-syntax, v0.3.0
steam-pirate-ui, v0.5.1

# Dev
No dev packages
```

**Comment by Osmose** [comment]
Thanks for the report!

I think this was fixed by 534c93799e6ea01d4c92c12f586c3ee750a65a15 and shipped in v0.1.2, can you update and verify?

**Comment by mikecluck** [comment]
Looks good. Thanks for the fix!
